Document custom mime type changes for 2.3.0 (#1295)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/dataweave-formats.adoc

Transform Message component or a Write operation to `output application/json` or
`output application/csv`.

> Starting in Mule 4.3.0, you can set the `output` directive using the format ID alone,
> instead of using the MIME type. For example, you might set the `output` header directive

Reviewer: I'm not sure I understand the use of "alone" and "instead of" together. I think we need whichever of the following is technically accurate:
```suggestion
Starting in Mule 4.3.0, you can set the `output` directive using the format ID alone,
without the MIME type. For example, you might set the `output` header directive
```
OR
```suggestion
Starting in Mule 4.3.0, you can set the `output` directive using the format ID instead of the MIME type. For example, you might set the `output` header directive
```

Reviewer 2: The way it used to be done was to use the MIME type and that indicated what format to use. Now, they can use the format ID instead of the MIME type.
